Domain Connect Spec Draft.adoc

```diff
@@ -622,31 +704,31 @@ be used when applying the template.
 
 |*Provider Name*
 |providerName
-|(optional) This parameter allows for the caller to provide additional text for display
+|(OPTIONAL) This parameter allows for the caller to provide additional text for display
 with the template providerName. This text should be used to augment the providerName value
 from the template, not replace it. This parameter is only allowed when the
 "sharedProviderName" attribute is set in the template. Note: this used to be controlled by the "shared"
 attribute in the template, which has been deprecated.
 
 |*Service Name*
 |serviceName
-|(optional) This parameter allows for the caller to provide additional text for display
+|(OPTIONAL) This parameter allows for the caller to provide additional text for display
 with the template serviceName. It should be used to augment the serviceName value
 from the template, not replace it. This parameter is only allowed when the
 "sharedServiceName" attribute is set in the template.
 
 |*Group Id*
 |groupId
-|(optional) This parameter specifies the groups from the template to apply. 
+|(OPTIONAL) This parameter specifies the groups from the template to apply. 

```

Comment:
Nit. There are some end of line white spaces that can be removed.

    sed -i 's/ *^M$/^M/' Domain\ Connect\ Spec\ Draft.adoc